lib/nexy/irc/plugins/roll.rb

```diff
@@ -9,7 +9,7 @@ class Roll < Base
     # Chooses a random number between 1 and 100
     match /roll/
     def execute(m)
-      rolled_number = roll()
+      rolled_number = roll
 
```

Comment:
Can you add an optional param for `max_roll_value` too?

The `arguments` method in the base class might be helpful for that. If no arg is passed in, then it should default to 100.